components/Footer/StyledNav.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const StyledNav = styled.div`
+  display: flex;
+  justify-content: space-around;
+  border-top: inset 1px #354f52;
```

Comment:
nice, I didn't know inset before. Wieder was gelernt ;)

Reply:
On my css-animation-day i spotted this :)